src/ufo.rs

```diff
@@ -28,16 +29,19 @@ static DEFAULT_LAYER_NAME: &str = "public.default";
 static DEFAULT_GLYPHS_DIRNAME: &str = "glyphs";
 static DEFAULT_METAINFO_CREATOR: &str = "org.linebender.norad";
 
+// groups and kerning: BTreeMap because we need sorting for serialization.
+pub type Groups = BTreeMap<String, Vec<String>>;
```

Comment:
I think it's worth being explicit about the structure of each map is.

Reply:
What do you mean?

Reply:
just a doc string I think? "A map of group names to the names of the glyphs in that group".